bleak/backends/p4android/scanner.py

```diff
@@ -276,13 +276,18 @@ def onScanResult(self, result):
             entry.getKey().toString(): bytes(entry.getValue())
             for entry in record.getServiceData().entrySet()
         }
+        if result.TX_POWER_NOT_PRESENT:
+            tx_power = None
+        else:
+            tx_power = result.getTxPower() # get Tx Power
```

Comment:
Based on the android docs, I think this need to be:
```suggestion
        tx_power = result.getTxPower()

        # change "not present" value to None to match other backends
        if tx_power == defs.ScanResult.TX_POWER_NOT_PRESENT:
            tx_power = None
```

And we will need to add `ScanResult` to the `defs` module.

```python
ScanResult = autoclass("android.bluetooth.le.ScanResult")
```

Reply:
We still need to add `ScanResult = autoclass("android.bluetooth.le.ScanResult")` to `defs.py`

Reply:
Thank you! Now I understand the structure of your code and usage of TX_POWER_NOT_PRESENT